modules/coreboot

```diff
@@ -150,6 +150,7 @@ endif
 
 $(coreboot_module)_configure := \
 	mkdir -p "$(build)/$(coreboot_dir)"; \
+	rm -rf $(build)/$(coreboot_dir)/*; \
```

Comment:
This poses the question if we should maybe move the coreboot board built artifacts out of coreboot build directory though

ie: this wipes builds/x86/coreboot-24.12/x230-maximized/* here.

Where shoul be coreboot board built artifacts prior of stitching a board rom if not under coreboot build directory?

This has considerations for CircleCI caching as well. We could move this out of build. Maybe tmp/$BOARD ?

That would require to modify `coreboot_dir` so where that would be
@JonathonHall-Purism insights?

Reply:
Was moved to dev cycle helper to clean coreboot_dir (build/ARCH/coreboot-VERSION/BOARD) and board_dir (build/ARCH/BOARD) to have reproducible builds